js/main.js

```diff
@@ -23,6 +23,88 @@ function handleSubmit(event) {
   data.nextEntryId += 1;
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  switchView('entries');
+  $ul.prepend(renderEntry(data.entries[0]));
+  noEntriesDiv.className = 'column-full no-entries hidden';
 }
 
 $form.addEventListener('submit', handleSubmit);
+
+function renderEntry(entry) {
+  var $list = document.createElement('li');
+  $list.setAttribute('class', 'column-full');
+
+  var rowDiv = document.createElement('div');
+  rowDiv.setAttribute('class', 'row');
+  $list.appendChild(rowDiv);
+
+  var imageDiv = document.createElement('div');
+  imageDiv.setAttribute('class', 'column-half display-image');
+  rowDiv.appendChild(imageDiv);
+
+  var $image = document.createElement('img');
+  $image.setAttribute('src', entry.url);
+  imageDiv.appendChild($image);
+
+  var columnHalfDiv = document.createElement('div');
+  columnHalfDiv.setAttribute('class', 'column-half');
+  rowDiv.appendChild(columnHalfDiv);
+
+  var $h2 = document.createElement('h2');
+  $h2.setAttribute('class', 'column-full');
+  var title = document.createTextNode(entry.title);
+  $h2.appendChild(title);
+  columnHalfDiv.appendChild($h2);
+
+  var $p = document.createElement('p');
+  $p.setAttribute('class', 'column-full entries-text');
+  var notes = document.createTextNode(entry.notes);
+  $p.appendChild(notes);
+  columnHalfDiv.appendChild($p);
+
+  return $list;
+}
+
+var $ul = document.querySelector('ul');
+var noEntriesDiv = document.querySelector('.no-entries');
+
+function handleDOMContentLoaded(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    $ul.appendChild(renderEntry(data.entries[i]));
+  }
+
+  if (data.entries.length === 0) {
+    noEntriesDiv.className = 'column-full no-entries';
+  } else {
+    noEntriesDiv.className = 'column-full no-entries hidden';
+  }
+
+  if (data.view === 'entry-form') {
+    switchView(data.view);
+  } else if (data.view === 'entries') {
+    switchView(data.view);
+  }
```

Comment:
Both of these conditional blocks do the exact same action, so no conditional is necessary